apps/main/src/dao/components/PageAnalytics/VeCrvFeesChart/FeesBarChart.tsx

```diff
@@ -36,7 +36,7 @@ const FeesBarChart = ({ data, height = 500 }: FeesBarChartProps) => (
         tick={{ fill: 'var(--page--text-color)', fontSize: 'var(--font-size-1)' }}
         tickLine={{ opacity: 0.3, strokeWidth: 0.3 }}
         axisLine={{ opacity: 0.3, strokeWidth: 0.3 }}
-        tickFormatter={(value) => `${formatNumber(value, { showDecimalIfSmallNumberOnly: true })}`}
+        tickFormatter={(value) => `${formatNumber(value, { ...(value > 10 && { decimals: 0 }) })}`}
```

Comment:
The `>10` is quite arbitrary and repetitive.
Why do we need it? What's special about these numbers?
 Can't we just abstract that away

Reply:
Hmm, perhaps we can just set decimals to 0 regardless of the size of value. This is btw how the `showDecimalIfSmallNumberOnly` property worked under the hood. But with proper abbreviations I think thos logic is no longer needed